postgres/src/test/java/org/jdbi/v3/postgres/TestSqlArrays.java

```diff
@@ -180,4 +185,36 @@ public void testIntList() throws Exception {
         @SqlUpdate(I_INSERT)
         void insertIntList(List<Integer> u);
     }
+
+    @Test
+    public void testWhereInArray() throws Exception {
+        WhereInDao dao = h.attach(WhereInDao.class);
+        dao.createTable();
+        Something a = new Something(1, "Alice");
+        Something b = new Something(2, "Bob");
+        Something c = new Something(3, "Candace");
+        Something d = new Something(4, "David");
+        Something e = new Something(5, "Emily");
+        dao.insert(a, b, c, d, e);
+
+        assertThat(dao.getByIds(1)).containsExactly(a);
+        assertThat(dao.getByIds(2)).containsExactly(b);
+        assertThat(dao.getByIds(3)).containsExactly(c);
+        assertThat(dao.getByIds(4)).containsExactly(d);
+        assertThat(dao.getByIds(5)).containsExactly(e);
+        assertThat(dao.getByIds(1, 2, 5)) // Three, sir!
+                .containsExactly(a, b, e);
+    }
+
+    @RegisterRowMapper(SomethingMapper.class)
+    public interface WhereInDao {
+        @SqlUpdate("create table something(id int, name text)")
+        void createTable();
+
+        @SqlBatch("insert into something(id, name) values (:id, :name)")
+        void insert(@BindBean Something... somethings);
+
+        @SqlQuery("select * from something where id = any(:ids) order by id")
+        List<Something> getByIds(int... ids);
```

Comment:
I threw this test in, wanted to see array arguments worked without having to use `@BindIn`